File: app/src/main/java/lnk/lastsubway/AboutActivity.java
package lnk.lastsubway;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

/**
 * Created by IGOR on 19.09.2015.
 */
public class AboutActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        TextView aboutContent = findViewById(R.id.about_content);
        aboutContent.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                finish();
            }
        });
    }
}

File: app/src/main/java/lnk/lastsubway/AboutActivity.kt
package lnk.lastsubway

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class AboutActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        val aboutContent = findViewById<TextView>(R.id.about_content)
        aboutContent.setOnClickListener { finish() }
    }
}
